tests/pipeline/test_pipeline_helper.py

```diff
@@ -355,3 +355,16 @@ def test_bad_outputs_mapping(self):
         pattern = "Outputs can't contain free inputs to the pipeline"
         with pytest.raises(ModularPipelineError, match=pattern):
             pipeline(raw_pipeline, outputs={"A": "C"})
+
+    def test_pipeline_always_copies(self):
+        original_pipeline = pipeline([node(constant_output, None, "A")])
+        new_pipeline = pipeline(original_pipeline)
+        assert new_pipeline is not original_pipeline
```

Comment:
To me it seems like the title of this test `test_pipeline_always_copies` doesn't fully match the `assert`. Perhaps add another `assert` that verifies that `new_pipeline` contains what we expect?

Reply:
Good point! Done in ea4e090.